ams/server/src/test/java/com/netease/arctic/server/optimizing/maintainer/TestAutoCreateIcebergTagAction.java

```diff
@@ -356,4 +388,17 @@ private void checkTag(Table table, String tagName, Snapshot snapshot) {
   private void checkSnapshots(Table table, int count) {
     Assert.assertEquals(Iterables.size(table.snapshots()), count);
   }
+
+  private long waitUntilAfter(long timestampMillis) {
+    long current = System.currentTimeMillis();
+    while (current <= timestampMillis) {
+      current = System.currentTimeMillis();
```

Comment:
Sleeping for a period of time rather than looping indefinitely seems better.

Reply:
My code is copy from `com.netease.arctic.server.optimizing.maintainer.TestSnapshotExpire`, and the while loop is also used directly in iceberg tester . But I don’t understand why you can’t just use sleep directly.

Reply:
I guess it may be because iceberg directly uses system.currentTimeMillis as the value to compare with snapshot when it expires. Will `Sleeping` as a native method cause some problems?

Reply:
got it.